2018.05.07 17:27:48 LOG5[ui]: stunnel 5.30 on x86_64-pc-linux-gnu platform
2018.05.07 17:27:48 LOG5[ui]: Compiled/running with OpenSSL 1.0.2g  1 Mar 2016
2018.05.07 17:27:48 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2018.05.07 17:27:48 LOG5[ui]: Reading configuration from file /home/tainp/remote/stunnel/dev_https
2018.05.07 17:27:48 LOG5[ui]: UTF-8 byte order mark not detected
2018.05.07 17:27:48 LOG5[ui]: FIPS mode disabled
2018.05.07 17:27:48 LOG4[ui]: Insecure file permissions on stunnel/stunnel.pem
2018.05.07 17:27:48 LOG5[ui]: Configuration successful
2018.05.07 17:28:48 LOG5[cron]: Updating DH parameters
2018.05.07 17:29:54 LOG5[ui]: stunnel 5.30 on x86_64-pc-linux-gnu platform
2018.05.07 17:29:54 LOG5[ui]: Compiled/running with OpenSSL 1.0.2g  1 Mar 2016
2018.05.07 17:29:54 LOG5[ui]: Threading:PTHREAD Sockets:POLL,IPv6,SYSTEMD TLS:ENGINE,FIPS,OCSP,PSK,SNI Auth:LIBWRAP
2018.05.07 17:29:54 LOG5[ui]: Reading configuration from file /home/tainp/remote/stunnel/dev_https
2018.05.07 17:29:54 LOG5[ui]: UTF-8 byte order mark not detected
2018.05.07 17:29:54 LOG5[ui]: FIPS mode disabled
2018.05.07 17:29:54 LOG4[ui]: Insecure file permissions on stunnel/stunnel.pem
2018.05.07 17:29:54 LOG5[ui]: Configuration successful
2018.05.07 17:30:37 LOG5[0]: Service [https] accepted connection from 116.100.242.108:32816
2018.05.07 17:30:37 LOG5[1]: Service [https] accepted connection from 116.100.242.108:32818
2018.05.07 17:30:37 LOG3[0]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:30:37 LOG5[0]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:30:37 LOG5[0]: Service [https] connected remote server from 127.0.0.1:49510
2018.05.07 17:30:37 LOG5[1]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:30:37 LOG5[1]: Service [https] connected remote server from 127.0.0.1:49512
2018.05.07 17:30:37 LOG5[0]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:30:37 LOG5[1]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:30:47 LOG5[2]: Service [https] accepted connection from 116.100.242.108:32900
2018.05.07 17:30:47 LOG5[3]: Service [https] accepted connection from 116.100.242.108:32898
2018.05.07 17:30:47 LOG3[2]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:30:47 LOG5[2]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:30:47 LOG5[2]: Service [https] connected remote server from 127.0.0.1:49518
2018.05.07 17:30:47 LOG5[3]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:30:47 LOG5[3]: Service [https] connected remote server from 127.0.0.1:49520
2018.05.07 17:30:47 LOG5[2]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:30:47 LOG5[3]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:30:54 LOG5[cron]: Updating DH parameters
2018.05.07 17:31:04 LOG5[cron]: DH parameters updated
2018.05.07 17:31:21 LOG5[4]: Service [https] accepted connection from 116.100.242.108:33220
2018.05.07 17:31:21 LOG3[4]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:31:21 LOG5[4]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:31:21 LOG5[4]: Service [https] connected remote server from 127.0.0.1:49524
2018.05.07 17:31:21 LOG3[4]: SSL_read: 14094418: error:14094418:SSL routines:ssl3_read_bytes:tlsv1 alert unknown ca
2018.05.07 17:31:21 LOG5[4]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:31:22 LOG5[5]: Service [https] accepted connection from 107.182.78.196:43904
2018.05.07 17:31:22 LOG5[5]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:31:22 LOG5[5]: Service [https] connected remote server from 127.0.0.1:49526
2018.05.07 17:31:22 LOG5[5]: Connection closed: 2312 byte(s) sent to SSL, 575 byte(s) sent to socket
2018.05.07 17:32:13 LOG5[6]: Service [https] accepted connection from 116.100.242.108:33806
2018.05.07 17:32:13 LOG5[7]: Service [https] accepted connection from 116.100.242.108:33808
2018.05.07 17:32:13 LOG3[6]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:32:13 LOG5[6]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:13 LOG5[6]: Service [https] connected remote server from 127.0.0.1:49536
2018.05.07 17:32:14 LOG5[6]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:14 LOG5[7]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:14 LOG5[7]: Service [https] connected remote server from 127.0.0.1:49538
2018.05.07 17:32:14 LOG5[7]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:16 LOG5[8]: Service [https] accepted connection from 116.100.242.108:33828
2018.05.07 17:32:16 LOG3[8]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:32:16 LOG5[8]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:16 LOG5[8]: Service [https] connected remote server from 127.0.0.1:49542
2018.05.07 17:32:16 LOG3[8]: SSL_read: Connection reset by peer (104)
2018.05.07 17:32:16 LOG5[8]: Connection reset: 438 byte(s) sent to SSL, 617 byte(s) sent to socket
2018.05.07 17:32:19 LOG5[9]: Service [https] accepted connection from 116.100.242.108:33874
2018.05.07 17:32:19 LOG5[10]: Service [https] accepted connection from 116.100.242.108:33872
2018.05.07 17:32:19 LOG5[9]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:19 LOG5[9]: Service [https] connected remote server from 127.0.0.1:49544
2018.05.07 17:32:19 LOG5[9]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:19 LOG5[10]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:19 LOG5[10]: Service [https] connected remote server from 127.0.0.1:49546
2018.05.07 17:32:19 LOG5[10]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:19 LOG5[11]: Service [https] accepted connection from 116.100.242.108:33876
2018.05.07 17:32:19 LOG5[11]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:19 LOG5[11]: Service [https] connected remote server from 127.0.0.1:49548
2018.05.07 17:32:22 LOG3[11]: SSL_read: Connection reset by peer (104)
2018.05.07 17:32:22 LOG5[11]: Connection reset: 528 byte(s) sent to SSL, 619 byte(s) sent to socket
2018.05.07 17:32:22 LOG5[12]: Service [https] accepted connection from 116.100.242.108:33902
2018.05.07 17:32:22 LOG5[12]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:22 LOG5[12]: Service [https] connected remote server from 127.0.0.1:49556
2018.05.07 17:32:22 LOG5[12]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:22 LOG5[13]: Service [https] accepted connection from 116.100.242.108:33904
2018.05.07 17:32:22 LOG5[13]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:22 LOG5[13]: Service [https] connected remote server from 127.0.0.1:49558
2018.05.07 17:32:22 LOG3[13]: SSL_read: Connection reset by peer (104)
2018.05.07 17:32:22 LOG5[13]: Connection reset: 2362 byte(s) sent to SSL, 562 byte(s) sent to socket
2018.05.07 17:32:30 LOG5[14]: Service [https] accepted connection from 116.100.242.108:33982
2018.05.07 17:32:30 LOG5[15]: Service [https] accepted connection from 116.100.242.108:33988
2018.05.07 17:32:30 LOG5[16]: Service [https] accepted connection from 116.100.242.108:33984
2018.05.07 17:32:30 LOG5[17]: Service [https] accepted connection from 116.100.242.108:33986
2018.05.07 17:32:30 LOG5[15]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:30 LOG5[14]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:30 LOG5[15]: Service [https] connected remote server from 127.0.0.1:49560
2018.05.07 17:32:30 LOG5[14]: Service [https] connected remote server from 127.0.0.1:49562
2018.05.07 17:32:30 LOG5[15]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:30 LOG5[14]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:30 LOG5[16]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:30 LOG5[16]: Service [https] connected remote server from 127.0.0.1:49564
2018.05.07 17:32:30 LOG5[16]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:30 LOG5[17]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:30 LOG5[17]: Service [https] connected remote server from 127.0.0.1:49566
2018.05.07 17:32:30 LOG5[17]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:30 LOG5[18]: Service [https] accepted connection from 116.100.242.108:33992
2018.05.07 17:32:30 LOG5[18]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:30 LOG5[18]: Service [https] connected remote server from 127.0.0.1:49568
2018.05.07 17:32:30 LOG5[18]: Connection closed: 438 byte(s) sent to SSL, 589 byte(s) sent to socket
2018.05.07 17:32:31 LOG5[19]: Service [https] accepted connection from 116.100.242.108:33994
2018.05.07 17:32:31 LOG5[19]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:31 LOG5[19]: Service [https] connected remote server from 127.0.0.1:49570
2018.05.07 17:32:31 LOG5[19]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:32:31 LOG5[20]: Service [https] accepted connection from 116.100.242.108:33996
2018.05.07 17:32:31 LOG5[20]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:32:31 LOG5[20]: Service [https] connected remote server from 127.0.0.1:49572
2018.05.07 17:32:34 LOG3[20]: SSL_read: Connection reset by peer (104)
2018.05.07 17:32:34 LOG5[20]: Connection reset: 528 byte(s) sent to SSL, 593 byte(s) sent to socket
2018.05.07 17:33:23 LOG5[21]: Service [https] accepted connection from 107.182.78.196:44203
2018.05.07 17:33:23 LOG5[21]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:33:23 LOG5[21]: Service [https] connected remote server from 127.0.0.1:49586
2018.05.07 17:33:23 LOG5[21]: Connection closed: 2312 byte(s) sent to SSL, 579 byte(s) sent to socket
2018.05.07 17:33:23 LOG5[22]: Service [https] accepted connection from 116.100.242.108:34482
2018.05.07 17:33:23 LOG5[23]: Service [https] accepted connection from 116.100.242.108:34484
2018.05.07 17:33:23 LOG5[24]: Service [https] accepted connection from 116.100.242.108:34486
2018.05.07 17:33:23 LOG5[22]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:33:23 LOG5[22]: Service [https] connected remote server from 127.0.0.1:49588
2018.05.07 17:33:23 LOG5[22]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:33:23 LOG5[23]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:33:23 LOG5[23]: Service [https] connected remote server from 127.0.0.1:49590
2018.05.07 17:33:23 LOG5[23]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:33:23 LOG5[24]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:33:23 LOG5[24]: Service [https] connected remote server from 127.0.0.1:49592
2018.05.07 17:33:23 LOG5[24]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:33:23 LOG5[25]: Service [https] accepted connection from 116.100.242.108:34496
2018.05.07 17:33:23 LOG5[25]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:33:23 LOG5[25]: Service [https] connected remote server from 127.0.0.1:49594
2018.05.07 17:33:23 LOG3[25]: SSL_read: Connection reset by peer (104)
2018.05.07 17:33:23 LOG5[25]: Connection reset: 438 byte(s) sent to SSL, 617 byte(s) sent to socket
2018.05.07 17:33:26 LOG5[26]: Service [https] accepted connection from 116.100.242.108:34526
2018.05.07 17:33:26 LOG5[27]: Service [https] accepted connection from 116.100.242.108:34528
2018.05.07 17:33:26 LOG5[28]: Service [https] accepted connection from 116.100.242.108:34524
2018.05.07 17:33:26 LOG5[27]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:33:26 LOG5[26]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:33:26 LOG5[27]: Service [https] connected remote server from 127.0.0.1:49596
2018.05.07 17:33:26 LOG5[26]: Service [https] connected remote server from 127.0.0.1:49598
2018.05.07 17:33:26 LOG5[27]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:33:26 LOG5[26]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:33:27 LOG5[28]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:33:27 LOG5[28]: Service [https] connected remote server from 127.0.0.1:49600
2018.05.07 17:33:27 LOG5[28]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:33:27 LOG5[29]: Service [https] accepted connection from 116.100.242.108:34530
2018.05.07 17:33:27 LOG5[29]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:33:27 LOG5[29]: Service [https] connected remote server from 127.0.0.1:49602
2018.05.07 17:33:29 LOG3[29]: SSL_read: Connection reset by peer (104)
2018.05.07 17:33:29 LOG5[29]: Connection reset: 528 byte(s) sent to SSL, 619 byte(s) sent to socket
2018.05.07 17:35:23 LOG5[30]: Service [https] accepted connection from 107.182.78.196:44516
2018.05.07 17:35:24 LOG5[30]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:35:24 LOG5[30]: Service [https] connected remote server from 127.0.0.1:49620
2018.05.07 17:35:24 LOG5[30]: Connection closed: 2312 byte(s) sent to SSL, 599 byte(s) sent to socket
2018.05.07 17:37:24 LOG5[31]: Service [https] accepted connection from 107.182.78.196:44788
2018.05.07 17:37:24 LOG5[31]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:24 LOG5[31]: Service [https] connected remote server from 127.0.0.1:49630
2018.05.07 17:37:25 LOG5[31]: Connection closed: 2312 byte(s) sent to SSL, 655 byte(s) sent to socket
2018.05.07 17:37:31 LOG5[32]: Service [https] accepted connection from 116.100.242.108:36530
2018.05.07 17:37:31 LOG3[32]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:37:31 LOG5[32]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:31 LOG5[32]: Service [https] connected remote server from 127.0.0.1:49636
2018.05.07 17:37:31 LOG5[32]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:31 LOG5[33]: Service [https] accepted connection from 116.100.242.108:36540
2018.05.07 17:37:32 LOG5[33]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:32 LOG5[33]: Service [https] connected remote server from 127.0.0.1:49638
2018.05.07 17:37:32 LOG3[33]: SSL_read: Connection reset by peer (104)
2018.05.07 17:37:32 LOG5[33]: Connection reset: 438 byte(s) sent to SSL, 589 byte(s) sent to socket
2018.05.07 17:37:32 LOG5[34]: Service [https] accepted connection from 116.100.242.108:36552
2018.05.07 17:37:32 LOG5[34]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:32 LOG5[34]: Service [https] connected remote server from 127.0.0.1:49640
2018.05.07 17:37:32 LOG5[34]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:32 LOG5[35]: Service [https] accepted connection from 116.100.242.108:36554
2018.05.07 17:37:32 LOG5[35]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:32 LOG5[35]: Service [https] connected remote server from 127.0.0.1:49642
2018.05.07 17:37:35 LOG3[35]: SSL_read: Connection reset by peer (104)
2018.05.07 17:37:35 LOG5[35]: Connection reset: 528 byte(s) sent to SSL, 593 byte(s) sent to socket
2018.05.07 17:37:52 LOG5[36]: Service [https] accepted connection from 116.100.242.108:36728
2018.05.07 17:37:52 LOG5[36]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:52 LOG5[36]: Service [https] connected remote server from 127.0.0.1:49654
2018.05.07 17:37:52 LOG5[36]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:52 LOG5[37]: Service [https] accepted connection from 116.100.242.108:36736
2018.05.07 17:37:52 LOG5[37]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:52 LOG5[37]: Service [https] connected remote server from 127.0.0.1:49656
2018.05.07 17:37:52 LOG3[37]: SSL_read: Connection reset by peer (104)
2018.05.07 17:37:52 LOG5[37]: Connection reset: 438 byte(s) sent to SSL, 589 byte(s) sent to socket
2018.05.07 17:37:52 LOG5[38]: Service [https] accepted connection from 116.100.242.108:36740
2018.05.07 17:37:53 LOG5[38]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:53 LOG5[38]: Service [https] connected remote server from 127.0.0.1:49658
2018.05.07 17:37:53 LOG5[38]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:53 LOG5[39]: Service [https] accepted connection from 116.100.242.108:36750
2018.05.07 17:37:53 LOG5[39]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:53 LOG5[39]: Service [https] connected remote server from 127.0.0.1:49660
2018.05.07 17:37:55 LOG5[39]: Connection closed: 14344 byte(s) sent to SSL, 593 byte(s) sent to socket
2018.05.07 17:37:55 LOG5[40]: Service [https] accepted connection from 116.100.242.108:36768
2018.05.07 17:37:55 LOG5[41]: Service [https] accepted connection from 116.100.242.108:36772
2018.05.07 17:37:55 LOG5[42]: Service [https] accepted connection from 116.100.242.108:36770
2018.05.07 17:37:55 LOG5[40]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:55 LOG5[40]: Service [https] connected remote server from 127.0.0.1:49668
2018.05.07 17:37:55 LOG5[40]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:55 LOG5[41]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:55 LOG5[41]: Service [https] connected remote server from 127.0.0.1:49670
2018.05.07 17:37:55 LOG5[41]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:55 LOG5[42]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:55 LOG5[42]: Service [https] connected remote server from 127.0.0.1:49672
2018.05.07 17:37:55 LOG5[42]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:55 LOG5[43]: Service [https] accepted connection from 116.100.242.108:36776
2018.05.07 17:37:55 LOG5[44]: Service [https] accepted connection from 116.100.242.108:36778
2018.05.07 17:37:55 LOG5[45]: Service [https] accepted connection from 116.100.242.108:36780
2018.05.07 17:37:55 LOG5[43]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:55 LOG5[43]: Service [https] connected remote server from 127.0.0.1:49674
2018.05.07 17:37:55 LOG5[44]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:55 LOG5[44]: Service [https] connected remote server from 127.0.0.1:49676
2018.05.07 17:37:55 LOG5[45]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:55 LOG5[45]: Service [https] connected remote server from 127.0.0.1:49678
2018.05.07 17:37:55 LOG5[43]: Connection closed: 3833 byte(s) sent to SSL, 546 byte(s) sent to socket
2018.05.07 17:37:55 LOG5[45]: Connection closed: 2311 byte(s) sent to SSL, 551 byte(s) sent to socket
2018.05.07 17:37:56 LOG5[44]: Connection closed: 352654 byte(s) sent to SSL, 548 byte(s) sent to socket
2018.05.07 17:37:56 LOG5[46]: Service [https] accepted connection from 116.100.242.108:36790
2018.05.07 17:37:56 LOG5[46]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:56 LOG5[46]: Service [https] connected remote server from 127.0.0.1:49680
2018.05.07 17:37:56 LOG5[46]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:56 LOG5[47]: Service [https] accepted connection from 116.100.242.108:36792
2018.05.07 17:37:56 LOG5[48]: Service [https] accepted connection from 116.100.242.108:36796
2018.05.07 17:37:56 LOG5[47]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:56 LOG5[47]: Service [https] connected remote server from 127.0.0.1:49682
2018.05.07 17:37:56 LOG5[47]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:56 LOG5[48]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:56 LOG5[48]: Service [https] connected remote server from 127.0.0.1:49684
2018.05.07 17:37:56 LOG5[49]: Service [https] accepted connection from 116.100.242.108:36798
2018.05.07 17:37:56 LOG5[48]: Connection closed: 2311 byte(s) sent to SSL, 551 byte(s) sent to socket
2018.05.07 17:37:56 LOG5[49]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:56 LOG5[49]: Service [https] connected remote server from 127.0.0.1:49686
2018.05.07 17:37:56 LOG5[49]: Connection closed: 136138 byte(s) sent to SSL, 560 byte(s) sent to socket
2018.05.07 17:37:56 LOG5[50]: Service [https] accepted connection from 116.100.242.108:36800
2018.05.07 17:37:56 LOG5[51]: Service [https] accepted connection from 116.100.242.108:36802
2018.05.07 17:37:56 LOG5[50]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:56 LOG5[50]: Service [https] connected remote server from 127.0.0.1:49688
2018.05.07 17:37:56 LOG5[50]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:56 LOG5[51]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:56 LOG5[51]: Service [https] connected remote server from 127.0.0.1:49690
2018.05.07 17:37:56 LOG5[51]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:56 LOG5[52]: Service [https] accepted connection from 116.100.242.108:36804
2018.05.07 17:37:56 LOG5[53]: Service [https] accepted connection from 116.100.242.108:36806
2018.05.07 17:37:56 LOG5[53]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:56 LOG5[53]: Service [https] connected remote server from 127.0.0.1:49692
2018.05.07 17:37:56 LOG5[52]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:56 LOG5[52]: Service [https] connected remote server from 127.0.0.1:49694
2018.05.07 17:37:57 LOG5[53]: Connection closed: 4118 byte(s) sent to SSL, 580 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[54]: Service [https] accepted connection from 116.100.242.108:36816
2018.05.07 17:37:57 LOG5[54]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[54]: Service [https] connected remote server from 127.0.0.1:49696
2018.05.07 17:37:57 LOG5[54]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[52]: Connection closed: 44137 byte(s) sent to SSL, 577 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[55]: Service [https] accepted connection from 116.100.242.108:36818
2018.05.07 17:37:57 LOG5[55]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[55]: Service [https] connected remote server from 127.0.0.1:49698
2018.05.07 17:37:57 LOG5[56]: Service [https] accepted connection from 116.100.242.108:36820
2018.05.07 17:37:57 LOG5[57]: Service [https] accepted connection from 116.100.242.108:36822
2018.05.07 17:37:57 LOG5[56]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[55]: Connection closed: 4118 byte(s) sent to SSL, 580 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[56]: Service [https] connected remote server from 127.0.0.1:49700
2018.05.07 17:37:57 LOG5[56]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[57]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[57]: Service [https] connected remote server from 127.0.0.1:49702
2018.05.07 17:37:57 LOG5[57]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[58]: Service [https] accepted connection from 116.100.242.108:36824
2018.05.07 17:37:57 LOG5[60]: Service [https] accepted connection from 116.100.242.108:36826
2018.05.07 17:37:57 LOG5[59]: Service [https] accepted connection from 116.100.242.108:36828
2018.05.07 17:37:57 LOG5[58]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[58]: Service [https] connected remote server from 127.0.0.1:49704
2018.05.07 17:37:57 LOG5[58]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[59]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[59]: Service [https] connected remote server from 127.0.0.1:49706
2018.05.07 17:37:57 LOG5[60]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[60]: Service [https] connected remote server from 127.0.0.1:49708
2018.05.07 17:37:57 LOG5[61]: Service [https] accepted connection from 116.100.242.108:36830
2018.05.07 17:37:57 LOG5[59]: Connection closed: 1872 byte(s) sent to SSL, 355 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[60]: Connection closed: 4118 byte(s) sent to SSL, 580 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[61]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[61]: Service [https] connected remote server from 127.0.0.1:49710
2018.05.07 17:37:57 LOG5[62]: Service [https] accepted connection from 116.100.242.108:36832
2018.05.07 17:37:57 LOG5[61]: Connection closed: 4118 byte(s) sent to SSL, 580 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[62]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[62]: Service [https] connected remote server from 127.0.0.1:49712
2018.05.07 17:37:57 LOG5[62]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[63]: Service [https] accepted connection from 116.100.242.108:36834
2018.05.07 17:37:57 LOG5[64]: Service [https] accepted connection from 116.100.242.108:36836
2018.05.07 17:37:57 LOG5[63]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[63]: Service [https] connected remote server from 127.0.0.1:49714
2018.05.07 17:37:57 LOG5[63]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[64]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[64]: Service [https] connected remote server from 127.0.0.1:49716
2018.05.07 17:37:57 LOG5[64]: Connection closed: 4118 byte(s) sent to SSL, 580 byte(s) sent to socket
2018.05.07 17:37:57 LOG5[65]: Service [https] accepted connection from 116.100.242.108:36838
2018.05.07 17:37:57 LOG5[65]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:37:57 LOG5[65]: Service [https] connected remote server from 127.0.0.1:49718
2018.05.07 17:37:58 LOG5[65]: Connection closed: 18920 byte(s) sent to SSL, 483 byte(s) sent to socket
2018.05.07 17:38:04 LOG5[66]: Service [https] accepted connection from 116.100.242.108:36896
2018.05.07 17:38:04 LOG5[66]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:38:04 LOG5[66]: Service [https] connected remote server from 127.0.0.1:49720
2018.05.07 17:38:04 LOG5[66]: Connection closed: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:38:04 LOG5[67]: Service [https] accepted connection from 116.100.242.108:36898
2018.05.07 17:38:04 LOG5[67]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:38:04 LOG5[67]: Service [https] connected remote server from 127.0.0.1:49722
2018.05.07 17:38:04 LOG3[67]: SSL_read: Connection reset by peer (104)
2018.05.07 17:38:04 LOG5[67]: Connection reset: 187 byte(s) sent to SSL, 859 byte(s) sent to socket
2018.05.07 17:39:25 LOG5[68]: Service [https] accepted connection from 107.182.78.196:45200
2018.05.07 17:39:25 LOG5[68]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:39:25 LOG5[68]: Service [https] connected remote server from 127.0.0.1:49730
2018.05.07 17:39:25 LOG5[68]: Connection closed: 2312 byte(s) sent to SSL, 595 byte(s) sent to socket
2018.05.07 17:41:26 LOG5[69]: Service [https] accepted connection from 107.182.78.196:45672
2018.05.07 17:41:26 LOG5[69]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:41:26 LOG5[69]: Service [https] connected remote server from 127.0.0.1:49742
2018.05.07 17:41:26 LOG5[69]: Connection closed: 2312 byte(s) sent to SSL, 631 byte(s) sent to socket
2018.05.07 17:43:26 LOG5[70]: Service [https] accepted connection from 107.182.78.196:46251
2018.05.07 17:43:27 LOG3[70]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:43:27 LOG5[70]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:43:27 LOG5[70]: Service [https] connected remote server from 127.0.0.1:49756
2018.05.07 17:43:27 LOG5[70]: Connection closed: 2312 byte(s) sent to SSL, 615 byte(s) sent to socket
2018.05.07 17:45:27 LOG5[71]: Service [https] accepted connection from 107.182.78.196:47356
2018.05.07 17:45:27 LOG5[71]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:45:27 LOG5[71]: Service [https] connected remote server from 127.0.0.1:49768
2018.05.07 17:45:28 LOG5[71]: Connection closed: 2312 byte(s) sent to SSL, 631 byte(s) sent to socket
2018.05.07 17:47:28 LOG5[72]: Service [https] accepted connection from 107.182.78.196:47757
2018.05.07 17:47:28 LOG5[72]: s_connect: connected 127.0.0.1:8001
2018.05.07 17:47:28 LOG5[72]: Service [https] connected remote server from 127.0.0.1:49782
2018.05.07 17:47:28 LOG5[72]: Connection closed: 2312 byte(s) sent to SSL, 607 byte(s) sent to socket
2018.05.07 17:49:29 LOG5[73]: Service [https] accepted connection from 107.182.78.196:48097
2018.05.07 17:49:29 LOG3[73]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 17:49:29 LOG3[73]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:49:29 LOG5[73]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:51:30 LOG5[74]: Service [https] accepted connection from 107.182.78.196:48581
2018.05.07 17:51:30 LOG5[74]: persistence: No cached address found
2018.05.07 17:51:30 LOG3[74]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:51:30 LOG3[74]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 17:51:30 LOG5[74]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:53:31 LOG5[75]: Service [https] accepted connection from 107.182.78.196:49508
2018.05.07 17:53:31 LOG5[75]: persistence: No cached address found
2018.05.07 17:53:31 LOG3[75]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 17:53:31 LOG3[75]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:53:31 LOG5[75]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:55:31 LOG5[76]: Service [https] accepted connection from 107.182.78.196:50127
2018.05.07 17:55:32 LOG3[76]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:55:32 LOG3[76]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 17:55:32 LOG5[76]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:57:19 LOG5[77]: Service [https] accepted connection from 50.116.7.184:36702
2018.05.07 17:57:19 LOG3[77]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 17:57:19 LOG3[77]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:57:19 LOG5[77]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:57:27 LOG5[78]: Service [https] accepted connection from 116.100.242.108:46660
2018.05.07 17:57:27 LOG3[78]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:57:27 LOG3[78]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 17:57:27 LOG5[78]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:57:32 LOG5[79]: Service [https] accepted connection from 107.182.78.196:50649
2018.05.07 17:57:33 LOG5[79]: persistence: No cached address found
2018.05.07 17:57:33 LOG3[79]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 17:57:33 LOG3[79]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:57:33 LOG5[79]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 17:59:33 LOG5[80]: Service [https] accepted connection from 107.182.78.196:51121
2018.05.07 17:59:33 LOG5[80]: persistence: No cached address found
2018.05.07 17:59:33 LOG3[80]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 17:59:33 LOG3[80]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 17:59:33 LOG5[80]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 18:01:34 LOG5[81]: Service [https] accepted connection from 107.182.78.196:51608
2018.05.07 18:01:34 LOG3[81]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 18:01:34 LOG3[81]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 18:01:34 LOG5[81]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 18:01:54 LOG5[82]: Service [https] accepted connection from 116.100.242.108:49390
2018.05.07 18:01:54 LOG3[82]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 18:01:54 LOG3[82]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 18:01:54 LOG5[82]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 18:03:35 LOG5[83]: Service [https] accepted connection from 107.182.78.196:52218
2018.05.07 18:03:35 LOG5[83]: persistence: No cached address found
2018.05.07 18:03:35 LOG3[83]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 18:03:35 LOG3[83]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 18:03:35 LOG5[83]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 18:05:36 LOG5[84]: Service [https] accepted connection from 107.182.78.196:52860
2018.05.07 18:05:36 LOG5[84]: persistence: No cached address found
2018.05.07 18:05:36 LOG3[84]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 18:05:36 LOG3[84]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 18:05:36 LOG5[84]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
2018.05.07 18:07:37 LOG5[85]: Service [https] accepted connection from 107.182.78.196:53334
2018.05.07 18:07:37 LOG3[85]: s_connect: connect 127.0.0.1:8001: Connection refused (111)
2018.05.07 18:07:37 LOG3[85]: s_connect: connect ::1:8001: Connection refused (111)
2018.05.07 18:07:37 LOG5[85]: Connection reset: 0 byte(s) sent to SSL, 0 byte(s) sent to socket
